go/oasis-test-runner/oasis/entity.go

```diff
@@ -18,7 +18,6 @@ import (
 const entityIdentitySeedTemplate = "oasis entity %d"
 
 var entityArgsDebugTest = []string{
-	"--" + flags.CfgDebugDontBlameOasis,
```

Comment:
This flag is now set in `oasis-test-runner/oasis/network.go` because it always needs to be enabled for test-runner networks, regardless if it contains the debug test entity or not.